python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -105,7 +105,8 @@ def __copy__(self, memo=None):
             else:
                 this_dict[elem] = None
         #obj.__setstate__(this_dict)
-        obj.getParent(self._getParent())
+        if self._getParent() is not None:
+            obj._setParent(self._getParent())
```

Comment:
Fair point - I had basically blindly copied this from `__deepcopy__` but now I think about it, `obj` wouldn't have a parent set and so it should be `None` anyway. I'll remove the additional check!